cypress/integration/pages/frontPage/tests.js

```diff
@@ -83,6 +85,28 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
               .should('be.visible');
           });
         });
+
+        if (serviceHasInfoAlsos(service)) {
+          it('should contain Index Alsos at a mobile view', () => {
+            cy.viewport('iphone-5');
+            cy.get('section li')
+              .eq(0)
+              .within(() => {
+                cy.get('div div div')
```

Comment:
`div div div` is very dependant on dom structure using the `ul` and `li` elements inside the `'section li'[0]` seems better imo